tests/external_executor/run_executor.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+from argparse import ArgumentParser
+import sys
+
+sys.path.append("/workspaces/CCF/tests")
```

Comment:
Let's not hardcode that.

Reply:
https://github.com/microsoft/CCF/pull/5018/commits/d1f01bc62346de78f95924b0af3b0c1bbe06d236